Automattic/wp-calypso
# Update help center icon

Related to # Automattic/dotcom-forge#7185

## Proposed Changes

* Update the SVGs used for help center to match the icon we use for it in the global sidebar.
* Updates the icon component used by the global nav sidebar to the consolidated HelpIcon component that also leverages the notification indicator state.

AFTER

Calypso masterbar
<img width="328" alt="Screenshot 2024-05-24 at 3 33 54 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/02c1079b-c157-4d44-8692-92f8502645f0">

with notif:
<img width="381" alt="Screenshot 2024-05-24 at 3 53 16 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/dbd285ee-b822-4e3e-a111-bef8e965d682">


Editor
<img width="316" alt="Screenshot 2024-05-24 at 3 32 51 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/9ae3d405-93a5-40cc-8167-32fdadf26c1a">


Wp-admin
<img width="284" alt="Screenshot 2024-05-24 at 3 34 17 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/9fb97003-bf76-4f22-85b2-80d11dd69154">

wp-admin classic
<img width="337" alt="Screenshot 2024-05-24 at 3 46 43 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/122166da-cb50-42ba-8685-227829c19b81">


 
## Why are these changes being made?


* We started by updating the icon in the global nav previously, and are now working to make this consistent across the wpcom experience.

## Testing Instructions




Calypso global nav:
* Visit calypso masterbar on this branch. Verify there are no changes to the icon. This should look the same, but we changed the underlying component so it is worth a check.

Calypso masterbar:

* Run this branch and visit calypso my home and ensure the help center icon is updated to match that of the global nav.
    * With notification:  I wasn't entirely sure how to trigger this to occur so I tested by changing [this condition locally to be true](https://github.com/Automattic/wp-calypso/blob/ebc69a78b6336163625daaa139946e21c666161e/packages/help-center/src/components/help-icon.tsx#L30). From there you can see the with notif version of the icon appear in the calypso masterbar.
 
Editor and wp-admin
*  install `install-plugin.sh $pluginSlug update/help-center-icon` on your sandbox
* sandbox widgets.wp.com and a test site url
* visit the editor and wp-admin and verify the new icon is present and help center works as expected.

Atomic
* find the calypso apps build job in the checks at the bottom of this PR and follow that link.
* in TC, goto the 'artificats' tab and download the .zip build artifact for editing toolkit.
* install this plugin on an atomic site. Deactivate the old version of ETK and activate this version.
* Verify the help center icon is changed and helpcenter works as expected.

## Pre-merge Checklist
